src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -16551,6 +16605,30 @@ instruct array_equalsC(iRegP_R1 ary1, iRegP_R2 ary2, iRegI_R0 result,
   ins_pipe(pipe_class_memory);
 %}
 
+instruct arrays_hashcode(iRegP_R1 ary, iRegI_R2 cnt, iRegI_R0 result, immI basic_type,
```

Comment:
I believe this is because the Java class that provides the method is called `Java.util.Arrays`. Additionally, `MacroAssembler` [declares](https://github.com/openjdk/jdk/blob/master/src/hotspot/cpu/aarch64/macroAssembler_aarch64.hpp#L1439) `arrays_equals` using the plural form as well.

Systematically changing these names would impact methods which are not directly related to this PR, namely `arrays_equals`, as well as other architectures beside AArch64. I'd suggest to do it in a separate PR in the future (if at all) to keep the focus of this one on `VectorizedHashCode` on AArch64, as indicated by the title.

Reply:
Ah, yes, that makes sense. Ok, so ignore this.